loginserver.js

```diff
@@ -53,10 +53,22 @@ var LoginServer = module.exports = (function () {
 	LoginServer.request = function (action, data, callback) {
 		return getLoginServer(action).request(action, data, callback);
 	};
+	var TimeoutError = LoginServer.TimeoutError = function (message) {
```

Comment:
It's not a big deal, I just didn't expect login server errors to be common enough to warrant this sort of thing.